tests/integration/queries/tpch_test_cases/all_tpch_queries.py

```diff
@@ -0,0 +1,9 @@
+from .query_1 import TESTCASE as q1
+from .query_2 import TESTCASE as q2
+from .query_3 import TESTCASE as q3
+from .query_4 import TESTCASE as q4
+from .query_5 import TESTCASE as q5
+
+TPCH_QUERY_TESTS = (
+        q1 + q2 + q3 + q4 + q5
+)
```

Comment:
I think originally, it was because the queries include the entire substrait plan as well as the sql...but now that that's been separated out, it probably would make sense to consolidate the test cases into a single file.

Reply:
ahh, yeah, that makes sense -- you would've been scrolling for days to  get from one test to another